src/libs/SidebarUtils.ts

```diff
@@ -108,6 +108,13 @@ function setWithLimit<TKey, TValue>(map: Map<TKey, TValue>, key: TKey, value: TV
 // Variable to verify if ONYX actions are loaded
 let hasInitialReportActions = false;
 
+type ReportActionsCount = Record<
+    string,
+    {
+        reportActionsCount: number;
+    }
+>;
```

Comment:
Can we simplify this to the following? We don't need a `reportActionsCount` key
```suggestion
type ReportActionsCount = Record<string, number>;
```